install/0000_30_machine-api-operator_00_credentials-request.yaml

```diff
@@ -64,3 +64,22 @@ spec:
     apiVersion: cloudcredential.openshift.io/v1
     kind: OpenStackProviderSpec
 ---
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: openshift-machine-api-gcp
+  namespace: openshift-cloud-credential-operator
+spec:
+  secretRef:
+    name: gcp-cloud-credentials
+    namespace: openshift-machine-api
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: GCPProviderSpec
+    predefinedRoles:
+    - "compute.instanceAdmin.v1"
```

Comment:
Upstream's roles:

https://github.com/kubernetes-sigs/cluster-api-provider-gcp/blob/master/pkg/cloud/google/serviceaccount.go#L57

Reply:
I doubt we need `compute.securityAdmin` and `iam.serviceAccountActor` is deprecated https://cloud.google.com/iam/docs/faq#in_what_scenarios_would_i_use_the_service_account_actor_role